### PR TITLE
Add minified bundle

### LINKED ISSUE #16: Add a minified version
Build up the code into a minified version.

Will also probably output built files into a `/dist` folder

### PR DESCRIPTION
**Pull request by nesvand**
Resolves #16 by adding a custom build file that handles building out the full and minified version, along with watch features to keep "serve" working.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,10 @@
   },
   "author": "Benjamin Plouzennec <plouzennec.benjamin@gmail.com>",
   "license": "GNU GPL",
-  "dependencies": {},
+  "dependencies": {
+    "rollup-plugin-uglify": "^2.0.1",
```
Comment (Okazari) [inline package.json]: Shoudn't theses belong in the devDependencies ? Till the built library wont use them ?

**Comment by nesvand** [comment]
Given that there are no tests written and iojs is effectively deprecated, I'd like to drop it from the TravisCI, but I'll obviously defer to you ;)

**Comment by Okazari** [comment]
Thanks for the PR ! :scream: you found that travis is there just for the badge :scream:

**Comment by Okazari** [comment]
Will merge and publish asap, thanks again :+1:

**Comment by nesvand** [comment]
Haha all good - maybe spin up an issue ticket as a reminder to create some tests ;)

**Comment by nesvand** [comment]
Note I version bumped to 2.1.0 if that makes sense to you (new feature, no API breaking changes) - I did kind of forget to update the documentation in this PR. I need to pop out quickly, but if you can hold off on the merge I'll fix the docs ;)

**Comment by Okazari** [comment]
I was planning to modify my test with "echo \"Test = doubt\" && exit 0" but it may be smartest to add some test :p (Even if i have no clue how to test it properly, this is about dom manipulation & data given by an audio file. I should at least unit test every testable function)
I'll wait for your update before merging then. 
Could you move the two new dependencies to devDependencies btw ?

File: package.json [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
+    "start": "concurrently --kill-others  \"npm run serve\" \"npm run build -w\"",
     "serve": "gulp serve",
     "prepublish": "npm run build",
     "test": "echo \"No test specified\" && exit 0",
-    "build": "rollup -c"
+    "build": "node build.js"
   },
   "author": "Benjamin Plouzennec <plouzennec.benjamin@gmail.com>",
   "license": "GNU GPL",
-  "dependencies": {
```
Comment (nesvand) [inline package.json]: @Okazari Just tagging you 'cause you keep missing that I fixed this in the second commit of my PR ;)

Comment (Okazari) [reply package.json]: Right i'm blind :scream:

**Pull request merged**